src/defs/server.ts

```diff
@@ -1954,58 +1926,4 @@ tmpDef.returnType = "bool";
 tmpDef.args = ["string data [", "int length]"];
 tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpClear";
-tmpDef.description = "This function removes all text from the current HTML output.";
-tmpDef.returnType = "bool";
-tmpDef.args = [""];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpRequestLogin";
-tmpDef.description = "This function makes the user's browser show a 'basic authentication' login box.";
-tmpDef.returnType = "bool";
-tmpDef.args = [""];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpSetResponseCode";
-tmpDef.description = "This function sets the HTTP status code that will be sent for the current HTML page.";
-tmpDef.returnType = "bool";
-tmpDef.args = ["int code"];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpSetResponseCookie";
-tmpDef.description = "This function sets the value for the specified HTTP cookie of the current HTML page. ";
-tmpDef.returnType = "bool";
-tmpDef.args = ["string cookieName", "string cookieValue"];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpSetResponseHeader";
-tmpDef.description = "This function sets the value for the specified HTTP response header of the current HTML page.";
-tmpDef.returnType = "bool";
-tmpDef.args = ["string headerName", "string headerValue"];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
-
-tmpDef = new MTAFunction;
-tmpDef.label = "httpWrite";
-tmpDef.description = "This function adds text to the output of the current HTTP file of the HTTP interface. The function can only be used on parsed (i.e not marked as raw) HTTP pages.";
-tmpDef.returnType = "bool";
-tmpDef.args = ["string data [", "int length]"];
-tmpDef.argDescs = {};
-tmpDef.scriptSide = ScriptSide.Server;
-ServerDefinitions.push(tmpDef);
+ServerDefinitions.push(tmpDef);
```

Comment:
This is now missing a new line ;) Please correct.

Reply:
There needs to be an extra line at the end of the file?